lib/engine/game/g_18_ms.rb

```diff
@@ -137,6 +145,21 @@ def tile_lays(entity)
 
         [{ lay: true, upgrade: true }, { lay: :not_if_upgraded, upgrade: false }]
       end
+
+      private
+
+      def rust(train, salvage_value)
+        rusted_trains = trains.select { |t| !t.rusted || t.name == train }
+        return if rusted_trains.empty?
+
+        rusted_trains.each do |t|
+          @bank.spend(salvage_value, t.owner)
+          t.rust!
+        end
+
+        @log << "-- Event: #{rusted_trains.uniq.join(', ')} trains rust --"
+        @log << "Corporations received a salvage value of #{format_currency(salvage_value)} per rusted train"
```

Comment:
Corporations receive a rebate of

Reply:
Salvage is the term used in the 18MS rules